contracts/Stake/StakeManager.sol

```diff
@@ -307,6 +308,46 @@ contract StakeManager is Initializable, Ownable {
         emit Redelegated(_stashId, _updatedCluster);
     }
 
+    function splitStash(bytes32 _stashId, bytes32[] calldata _tokens, uint256[] calldata _amounts) external {
```

Comment:
don't think this should be external - the other functions are public which allow contracts to create stashes, but this wouldn't be accessible to contracts